aries-site/src/pages/whats-new.mdx

```diff
@@ -51,6 +51,37 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 19-23, 2021
+- Begun iterating Notifications taxonomy based on consuming of research results
+- Continued [ideating on writing guidance for Navigation] (https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20)
+- Audited all of Figma files to make sure we're using the correct MetricHPEXS font
+
+
+## July 12-16, 2021
+
+- Began Notifications Taxonomy Survey
+- Updated [Header colors on DateInput component in Figma](https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=0%3A1)
+- Removed [Selected state from Menu on Figma file](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1) to align with Design System site
+- Updated [Data Table with Search Value](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A242) to be updated to match the styling we show in Filtering Figma/on DS site.
+- [Updated Select dropdown items on Figma file to use regular text weight rather than form](https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/HPE-Select-Component?node-id=0%3A1)
+- Updated text styling on [drop list items for TextInput from form/value to text/default](https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component?node-id=0%3A1)
+- Enhanced the way Menu was built on Figma by making it [one component rather than a button with a list drop.](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1)
```

Comment:
```suggestion
- Enhanced the way Menu is built on Figma by making it [one component rather than a button with a list drop.](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1)
```